workflows/benchmarks/README.md

```diff
@@ -0,0 +1,15 @@
+## Docker
+
+Running the benchmark analysis notebook should be done via docker, with the following command to mount the local directory as the home direcory and pass in the current user's AWS credentials and launch RStudio.
```

Comment:
```suggestion
Running the benchmark analysis notebook should be done via docker, with the following command to mount the local directory as the home directory and pass in the current user's AWS credentials and launch RStudio.
```

Reply:
Can you add a brief reference to `.aws` like you have in `workflows/images/scpca-r/README.md` and/or to running `aws setup` ahead of time? Quoting you from your initial comment filing the PR:

> To run this notebook therefore requires that you already have run `aws setup` on your local system with credentials that allow read access to `s3://nextflow-ccdl-results`